### PR TITLE
Added separate component for the keyboard

### PR DESCRIPTION
**Pull request by Jsimpada**
### ```<Keyboard/>``` Component
Functions and logic for the keyboard were moved to the components folder. Volume and splash screen props are being passed down to the keyboard component.

image

### CODE REVIEW

File: src/components/keyboard/keyboard.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    }, [play, setKey, splash, keys]);
+  };
+
+function Keyboard(props) {
+    const ordered = ['$', 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'a', 'b', 'B', 'c', 'C', 'd', 'D', 'e', 'E',
+                     'f', 'g', 'G', 'h', 'H', 'i', 'I', 'j', 'J', 'k', 'l', 'L', 'm', '!', '%', '(',
+                     '*', '@', '^', 'n', 'o', 'O', 'p', 'P', 'q', 'Q', 'r', 's', 'S', 't', 'T', 'u',
+                     'v', 'V', 'w', 'W', 'x', 'y', 'Y', 'z', 'Z'];
+
+    const keyboard = ['1', '!', '2', '@', '3', '4', '$', '5', '%', '6', '^', '7', '8',
```
Comment (elijaholan) [inline src/components/keyboard/keyboard.js:42]: A bit redundant, could be refactored into one list. However, we would need to change the key samples.

File: src/routes/App.js [hunk 2]
```diff
@@ -96,25 +46,9 @@ function App() {
           <Metronome volume={volume}/>
           <Volume setVolume={setVolume} volume={volume}/>
         </div>
+        
+        <Keyboard setKey={setKey} volume={volume} splash={splash}/>
```
Comment (elijaholan) [inline src/routes/App.js:50]: A lot more concise, looks great!

File: src/components/keyboard/keyboard.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import useSound from 'use-sound';
+import { useEffect } from 'react';
+
+import keys from "../../sounds/keys.mp3";
+
+const useKeyboardBindings = (play, setKey, splash, keys) => {
```
Comment (brunogrcsada) [inline src/components/keyboard/keyboard.js:6]: Add a few comments to explain this function. Make each parameter clear!

**Review by brunogrcsada — APPROVED** [review]
Perfect refactoring! The keyboard component only contains code relevant to the keys and sound. Well done Jaheim :+1:

**Pull request merged**